src/components/Calendar/index.tsx

```diff
@@ -106,33 +89,48 @@ const Calendar: React.FC<CalendarProps> = ({
                 setMonth((prevMonth) => prevMonth + 1);
               }
             }}
-          >
-            next
-          </button>
+          />
+          <div className="w-1" />
         </div>
       </div>
-      <div className="grid grid-cols-7">
+      <div className="h-8"></div>
+      <div className="grid grid-cols-7 gap-x-1 gap-y-2.5 w-full">
         <React.Fragment>
           {weekdayNamesAbbreviated.map((dayOfWeek, i) => (
-            <div className="text-center select-none" key={i}>
+            <div className="text-center select-none font-bold" key={i}>
               {dayOfWeek.toUpperCase()}
```

Comment:
I like keeping the capitalized words in the data array (because toUpper and toLower are easy function calls), and I don't think it really affects performance, so I'll leave it in